### PR TITLE
Notepad++ Cheat Sheet: Add description

### PR DESCRIPTION
**Pull request by 122131**
###### What does your Pull Request do (check all that apply)?

- [ ] New Instant Answer
    - [ ] Cheat Sheets: **`New {Cheat Sheet Name} Cheat Sheet`**
    - [ ] Other: **`New {IA Name} Instant Answer`**
- [x] Improvement
    - [ ] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
    - [x] Enhancement: **`{Notepad++ Cheat Sheet
 }: {added description}`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**

###### Description of changes

Added description

---

Instant Answer Page: https://duck.co/ia/view/notepad_plus_plus_cheat_sheet

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @biscuitNinja

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by edgesince84** [comment]
👍

File: share/goodie/cheat_sheets/json/notepad-plus-plus.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
     "id": "notepad_plus_plus_cheat_sheet",
     "name": "Notepad++",
+    "description": "Notepad++ Default Keyboard Shortcuts",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/notepad-plus-plus.json]: nit: We should use sentence case: `Notepad++ default keyboard shortcuts`

File: share/goodie/cheat_sheets/json/notepad-plus-plus.json [hunk 2]
```diff
@@ -1,6 +1,7 @@
 {
     "id": "notepad_plus_plus_cheat_sheet",
     "name": "Notepad++",
+    "description": "Default Keyboard Shortcuts",
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/notepad-plus-plus.json]: For the description, please could you use lower case for everything except the first letter, for example:
"Notepad++ default keyboard shortcuts"

File: share/goodie/cheat_sheets/json/notepad-plus-plus.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
     "id": "notepad_plus_plus_cheat_sheet",
     "name": "Notepad++",
-    "description": "Default Keyboard Shortcuts",
+    "description": "Default keyboard shortcuts",
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/notepad-plus-plus.json]: Thanks - sorry, one more. Please could you include Notepad++, for example:
"Notepad++ default keyboard shortcuts" or
"Default keyboard shortcuts for Notepad++"

Comment (122131) [inline share/goodie/cheat_sheets/json/notepad-plus-plus.json]: Sure!

**Comment by tagawa** [comment]
Great, thank you @122131 - merging!

**Comment by edgesince84** [comment]
thanks again @122131! 🎉

**Pull request merged**